packages/types/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@suitest/types",
+  "version": "2.0.4",
```

Comment:
probably mistake?

Reply:
We are using single version for all packages in this repo. So, it's set to same value as in root package json. Will be updated when we are ready for release.